extensions/mssql/src/contracts.ts

```diff
@@ -31,6 +31,22 @@ export class TelemetryParams {
 
 // ------------------------------- </ Telemetry Sent Event > ----------------------------------
 
+// ------------------------------- < Security Token Request > ------------------------------------------
+export interface RequestSecurityTokenParams {
+	authority: string;
```

Comment:
What is authority here?

Reply:
The authority is the Identifier of the authority, a URL. It (and those other members) maps to the [KeyVaultClient.AuthenticationCallback Delegate](https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.keyvault.keyvaultclient.authenticationcallback?view=azure-dotnet-legacy) parameters.
An example of authority would be:
`{"authority": "https://login.windows.net/XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX"}`

Reply:
I think you should add Provider as well - so we know when this comes in is it a request for an azure account or {whateverService} account.

Reply:
Provider added.